### ISSUE TITLE
output + media_query + transport auto not working properly

### ISSUE THREAD

**Issue by nk-o** [themeum/kirki#1127]
### Issue description:

When I trying to add some css property to specific `@media` query, Kirki simply put it without using `@media`. This occurs only when css added without page reload. On this screenshot see how to added property:

![screen shot 2016-09-18 at 11 58 00](https://cloud.githubusercontent.com/assets/14810891/18614462/ce2743ce-7d97-11e6-988f-7a2d8fca04ae.png)

First property added when I try to change value in customizer (is not correct).
Second property added in php before customizer loaded (is correct).
### Version used:

2.3.6
### Using theme_mods or options?

theme_mods
### Code to reproduce the issue (config + field(s))

Example with typography field, but it's not working properly for all available field types. 

``` php
Kirki::add_field('nk-theme', array(
    'type' => 'typography',
    'settings' => 'typography_html',
    'label' => esc_attr__('HTML', 'nk-theme'),
    'section' => 'typography',
    'default' => array(
        'font-size' => '14px'
    ),
    'output' => array(
        array(
            'element' => 'html',
            'media_query' => '@media (min-width: 991px)'
        ),
    ),
    'transport' => 'auto'
));
```

**Comment by aristath** [comment]
Closing as duplicate of #1184

**Comment by aristath** [comment]
Fixed in 3.0
